cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -1227,6 +1231,52 @@ private boolean hasRebalanceAnnotation(KafkaRebalance kafkaRebalance) {
                 kafkaRebalance.getMetadata().getAnnotations().containsKey(ANNO_STRIMZI_IO_REBALANCE);
     }
 
+    private boolean shouldRenewRebalance(KafkaRebalance kafkaRebalance, KafkaRebalanceAnnotation rebalanceAnnotation) {
+        // check if request are made when CC is not active
+        if (kafkaRebalance.getStatus().getConditions().stream().anyMatch(condition -> "ConnectException".equals(condition.getReason()))) {
+            return true;
+        } else if (rebalanceAnnotation == KafkaRebalanceAnnotation.refresh) {
```

Comment:
why this is not in OR with the previous one? They both return `true`.

Reply:
Done @ppatierno